docker/devbox/README.md

```diff
@@ -73,6 +73,18 @@ nobody@2d25d207a7a7:/usr/wikia/slot1/current/src$ cd tests
 nobody@2d25d207a7a7:/usr/wikia/slot1/current/src/tests$ SERVER_DBNAME=firefly php run-test.php --stderr --configuration=phpunit.xml --exclude-group Infrastructure,Integration,ExternalIntegration,ContractualResponsibilitiesValidation
 nobody@2d25d207a7a7:/usr/wikia/slot1/current/src/tests$ SERVER_DBNAME=firefly php run-test.php --stderr --configuration=phpunit.xml --group Integration
 ```
+### Xdebug
+
+The Xdebug extension is enabled and configured to connect to port 9000 of the docker host. In order to use xdebug follow
+this article: https://wikia-inc.atlassian.net/wiki/spaces/EN/pages/35723807/Debugging+with+Xdebug+and+Charles. 
+
+In a nutshell:
+1. Forward port 9000 to your laptop by running this command in the terminal:
+    ```sh
+    ssh -o ServerAliveInterval=20 -R 9000:localhost:9000 dev-<devboxname>-18
+    ```
+2. Enable the `Listen for debug connections` button in PhpStorm
+3. Add `XDEBUG_SESSION_START=1` query parameter to the request
```

Comment:
I would add one caveat/gotcha that sources that are being reported by xdebug live in `/usr/wikia/slot1/current/` and not the path used on dev-box

Reply:
done